storable/output.py

```diff
@@ -0,0 +1,151 @@
+"""
+import storable.output
+storable.output.serialize({'x': 'bar', 'y': 1, 'z': 1.23, 'w':[], 'v':[1,2,3]})
+"""
+from struct import pack
+
+
+def serialize(py_jsonable, pst_prefix=True, version=(5, 9)):
+    ret_bytes = bytes()
+    if pst_prefix:
+        ret_bytes += b'pst0'
+    ret_bytes += bytes(version)
+    ret_bytes += process_item(py_jsonable)
+    return ret_bytes
+
+
+def unsigned_int(value, area=4):
+    """
+    Returns unsigned value with area param's byte-length
+    This is also used for writing out the size
+    """
+    final = []
+    for i in range(area-1, -1, -1):
+        digit = 256**i
+        dig_val = value // digit
+        final.append(dig_val)
+        value -= (dig_val * digit)
+    return bytes(final)
+
+
+def byte_len(size, area=4):
+    return unsigned_int(size, area=area)
+
+
+def signed_smallint(value):
+    """
+    Returns signed value
+    """
+    negative = (value < 0)
+    value = abs(value)
+    assert(value < 128)
```

Comment:
I would personally prefer a `ValueError` here. First, `assert` statements get dropped in "optimized mode" and have no helpful message associated with them. Both issues are resolved by throwing a `ValueError`.